src/main/java/org/apache/sling/junit/impl/servlet/TestLogServlet.java

```diff
@@ -98,18 +127,20 @@
 
     private Layout<ILoggingEvent> layout;
 
+    @SuppressWarnings("rawtypes")
     private ServiceRegistration filter;
 
     private volatile Description currentTest;
 
     private final Object appenderLock = new Object();
 
     @Activate
-    protected void activate(BundleContext ctx, Map<String, ?> config) throws Exception {
-        registerServlet(config);
-        registerAppender(config);
+    @Modified
+    protected void activate(BundleContext ctx, Config cfg) throws Exception {
```

Comment:
just pass the config as argument

Reply:
The use of @Modfied has been fixed.   Not removing ctx for it is used by registerFilter(ctx);